src/components/ui/toast.tsx

```diff
@@ -0,0 +1,115 @@
+'use client';
+
+import * as React from 'react';
+import * as ToastPrimitives from '@radix-ui/react-toast';
```

Comment:
I know it's very cumbersome to refactor, but could you replace these wildcard imports with named imports?